how-to/install/enable-network-role.rst

```diff
@@ -0,0 +1,98 @@
+Enable Network Role for Canonical Openstack
+===========================================
+
+
+This how-to guide provides all necessary information to deploy dedicated network nodes in your
+`Canonical OpenStack`_ deployment with Sunbeam.
+
+Overview
+--------
+
+By default, Sunbeam deploys OpenStack with combined control and compute nodes, where networking
+services run alongside compute workloads. The network role allows you to deploy dedicated network
+nodes that handle all networking functions separately.
+
+Network nodes run MicroOVN and the OpenStack network agents (deployed as a subordinate charm
+co-located with MicroOVN), providing:
+
+* OVN chassis functionality for overlay networking
+* Gateway services for North-South traffic
+* External network connectivity for instances
+* Remote connectivity for bare-metal instances when Ironic is enabled
+
+.. important ::
+
+   Network and compute roles are **mutually exclusive**. A node cannot be both a compute node
```

Comment:
That's not the reason why a node cannot be both network and compute.

It cannot be because, today, the compute role provides the networking features. That's the reason why they are mutually exclusives, as they would conflict.

In a future release, network will be completely decorrelated from compute, and they will no longer be mutually exclusive.